include/LightGBM/cuda/cuda_utils.h

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) 2020 IBM Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE file in the project root for license information.
+ */
```

Comment:
@guolinke is it ok to have an IBM copyright on this code? How does that work with our CLA (https://cla.opensource.microsoft.com/Microsoft/LightGBM\), which says the following:

> You grant Microsoft, and those who receive the Submission directly or indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in the Submission to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute the Submission and such derivative works, and to sublicense any or all of the foregoing rights to third parties.

Reply:
I think MIT license (https://github.com/microsoft/LightGBM/blob/master/LICENSE) meet these CLA statements. So there are no conflicts.

Reply:
ok sounds good, thanks